### PR TITLE
License apis

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: kp_service_test/src/test/resources/templates/license/v3/create/testCreateLicenseWithStatus/response.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "msgid": null,
     "err": "ERROR_RESTRICTED_PROP",
     "status": "failed",
-    "errmsg": "Properties in list [status] are not allowed in request"
+    "errmsg": "Properties in list List(status) are not allowed in request"
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/license/v3/create/testCreateLicenseWithStatus/response.json:10]: Please revert this change and fix the code.

File: kp_service_test/src/test/resources/templates/license/v3/read/testReadLicenseAfterRetire/response.json [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
-  "ver" : "3.0",
-   "ts": "@matchesDatePattern('yyyy-MM-dd')@",
-  "params" : {
-    "resmsgid" : null,
-    "msgid" : null,
-    "err" : "RESOURCE_NOT_FOUND",
-    "status" : "failed",
-    "errmsg" : "License not found with identifier: ${identifier}"
-  },
-  "responseCode" : "RESOURCE_NOT_FOUND",
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/license/v3/read/testReadLicenseAfterRetire/response.json:12]: Is it the functional requirement changes to provide Retired License data instead of throwing exception?

File: kp_service_test/src/test/resources/templates/license/v3/update/testUpdateLicenseWithIdentifier/response.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
         "msgid": null,
         "err": "ERROR_RESTRICTED_PROP",
         "status": "failed",
-        "errmsg": "Properties in list [status, identifier, name] are not allowed in request"
+        "errmsg": "Properties in list List(status, identifier, name) are not allowed in request"
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/license/v3/update/testUpdateLicenseWithIdentifier/response.json:10]: Please revert this change and fix the code.

File: kp_service_test/src/test/resources/templates/license/v3/update/testUpdateLicenseWithStatus/response.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
         "msgid": null,
         "err": "ERROR_RESTRICTED_PROP",
         "status": "failed",
-        "errmsg": "Properties in list [status, identifier, name] are not allowed in request"
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/license/v3/update/testUpdateLicenseWithStatus/response.json:10]: Please revert this change and fix the code.

File: kp_service_test/src/test/resources/templates/license/v3/read/testReadLicenseAfterRetire/response.json (hunk 2 above)
Comment (rheafernandes) [reply kp_service_test/src/test/resources/templates/license/v3/read/testReadLicenseAfterRetire/response.json:12]: This was the updated requirement I think. I need to confirm.
